test/BUILD.gn

```diff
@@ -1113,6 +1113,7 @@ test("brave_browser_tests") {
       "//brave/browser/ui/views/brave_actions/brave_actions_container_browsertest.cc",
       "//brave/browser/ui/views/brave_shields/cookie_list_opt_in_browsertest.cc",
       "//brave/browser/ui/views/frame/brave_non_client_hit_test_helper_browsertest.cc",
+      "//brave/browser/ui/views/frame/brave_system_menu_model_builder_browsertest.cc",
```

Comment:
we should avoid adding indivisual cc file here. You need to find proper browser_test target.
if not need to create it and adding it to deps here.

Reply:
oh, okay! Then I think we can add it in `browser_tests` target in `src/brave/browser/ui/BUILD.gn`